staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types_jsonschema.go

```diff
@@ -59,7 +59,7 @@ type JSONSchemaProps struct {
 // JSON represents any valid JSON value.
 // These types are supported: bool, int64, float64, string, []interface{}, map[string]interface{} and nil.
 type JSON struct {
-	Raw []byte `protobuf:"bytes,1,opt,name=raw"`
+	Raw []byte `json:"Raw,omitempty" protobuf:"bytes,1,opt,name=raw"`
```

Comment:
This should be lower case.

Reply:
If someone stores these in etcd in JSON, then the serialized data in etcd will already be Uppercase, changing it to lower case now will make their etcd data invalid.